strawberryfields/parameters.py

```diff
@@ -179,6 +179,9 @@ def do_evaluate(p):
         return ret[0]
     return ret
 
+# default printer for lambdify to use (we also use 'tensorflow' for TF objects)
+par_evaluate.lambdify_printer = ['numpy']
```

Comment:
How would this be modified to allow it to work with both TF and numpy? Currently it looks like something that must be set manually.

Reply:
Right now it is. I was planning to change it so that `par_evaluate` switches to the `tensorflow` printer if any of the `vals` are TF objects. I'm kinda leaning towards simply checking if `type(x).__module__` begins with `'tensorflow'`.

Reply:
:+1: Would be good to have this commented out in the code with a todo

Reply:
I can also change it right away, would that be better?

Reply:
yep, that would be perfect

Reply:
Done.